findbugs.properties

```diff
@@ -1,145 +1,125 @@
+2.4.mavenGroupId = org.codehaus.sonar-plugins.java
+3.0.description = <b>WARNING: This version requires Java 7 for execution.</b><br/>New version includes Findbugs 3.0.0 and brings support of Java 8 bytecode analysis.
+3.0.sqVersions = [4.5.1]
+3.0.downloadUrl = http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/3.0/sonar-findbugs-plugin-3.0.jar
+3.0.date = 2014-08-12
+3.0.changelogUrl = https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11762
+3.0.mavenGroupId = org.codehaus.sonar-plugins.java
+3.1.description = Rule Tags for owasp-top-10
+3.1.sqVersions = [4.5.1]
+3.1.downloadUrl = http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/3.1/sonar-findbugs-plugin-3.1.jar
+3.1.date = 2014-11-07
+3.1.changelogUrl = https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11763
+3.1.mavenGroupId = org.codehaus.sonar-plugins.java
+3.2.description = Integration of FindSecBugs library in version 1.3.0, upgrade to Findbugs 3.0.1 and 189 FindBugs rules which are now covered by the SonarQube Java plugin have been flagged as deprecated.
+3.2.sqVersions = [4.5.2,5.1]
+3.2.downloadUrl = http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/3.2/sonar-findbugs-plugin-3.2.jar
+3.2.date = 2015-03-17
+3.2.changelogUrl = https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11764
+3.2.mavenGroupId = org.codehaus.sonar-plugins.java
+3.3.description = Fix missing SQALE debt. Upgrade of FB-Contrib to 6.2.3. Upgrade of Find Security Bugs to version 1.4.2.
+3.3.sqVersions = [4.5.2,5.5]
+3.3.downloadUrl = https://binaries.sonarsource.com/Distribution/sonar-findbugs-plugin/sonar-findbugs-plugin-3.3.jar
+3.3.date = 2015-09-01
+3.3.changelogUrl = https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10950&version=11765
+3.4.3.description = Update to FB-Contrib 6.6.1, and FindSecBugs 1.4.6 FindBugs 3.0.1 rule descriptions, and compatibility with SonarQube Java Plugin 4.0
+3.4.3.sqVersions = [5.6,6.2]
+3.4.3.downloadUrl = https://github.com/SonarQubeCommunity/sonar-findbugs/releases/download/3.4/sonar-findbugs-plugin-3.4.3.jar
+3.4.3.date = 2015-07-11
+3.4.3.changelogUrl = https://github.com/SonarQubeCommunity/sonar-findbugs/releases/tag/3.4
+3.4.4.description = Bug fixes
+3.4.4.sqVersions = [5.6.5,6.3]
+3.4.4.downloadUrl = https://github.com/SonarQubeCommunity/sonar-findbugs/releases/download/3.4.4/sonar-findbugs-plugin-3.4.4.jar
+3.4.4.date = 2016-09-23
+3.4.4.changelogUrl = https://github.com/SonarQubeCommunity/sonar-findbugs/compare/3.4.3...3.4.4
+4.0.3.description = Use SpotBugs 4.2.0, sb-contrib 7.4.7, and findsecbugs 1.11.0
+4.0.3.sqVersions = [7.9,LATEST]
+4.0.3.date = 2021-01-02
+4.0.3.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.0.3
```

Comment:
Your version description says 'Use SpotBugs 4.2.0, sb-contrib 7.4.7, and findsecbugs 1.11.0' but your changelog link includes none of that.

Reply:
I've updated the page https://github.com/spotbugs/sonar-findbugs/releases/tag/4.0.3 thank you!